spec/requests/api/v1/passwords/create_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+# Test suite for POST /api/v1/auth/password
+describe 'POST /api/v1/auth/password', type: :request do
+  let(:user) { create(:user) }
+
+  before(:each) { user.confirm }
+
+  context 'when the request is valid' do
+    before do
+      post  '/api/v1/auth/password',
+            params: {
+              email: user.email,
+              redirect_url: 'https://www.google.com'
```

Comment:
I would use a random url here 🤔 Anyway I think we already discuss this and maybe there is a rebase missing or something like this